datacapturing/src/main/java/de/cyface/datacapturing/StartUpFinishedHandler.java

```diff
@@ -39,7 +39,7 @@
      *
      * @param measurementIdentifier The identifier of the measurement that is captured by the started capturing process.
      */
```

Comment:
This API changed recently. Implementing the new method resulted into a warning from the IDE that simple types (long) cannot have a NonNull flag. Thus, I removed the flag.